pybandits/strategy.py

```diff
@@ -221,11 +221,14 @@ def select_action(self, p: Dict[ActionId, Probability], actions: Dict[ActionId,
         # define the set of feasible actions
         feasible_actions = [a for a in p.keys() if p[a] >= (1 - self.subsidy_factor) * max_p]
 
-        # feasible action costs
-        _, cheapest_action = min((b.cost, a) for a, b in actions.items() if a in feasible_actions)
+        # feasible actions and their characteristics
+        sortable_actions = [(actions[a].cost, 1 - p[a], a) for a in feasible_actions]
+
+        # best action is the one with cheapest cost (+ higher proba in case of equality)
+        _, _, best_action = sorted(sortable_actions)[0]
```

Comment:
To use the same naming convention as other strategies, I would suggest renaming `best_action` to `selected_action`

```
# select the action with the cheapest cost (and the highest probability in case of cost equality)
 _, _, selected_action = sorted(sortable_actions)[0]
```